### PR TITLE
Adding date

### PR DESCRIPTION
**Pull request by julesnikolova**

### CODE REVIEW

File: .eslintcache [hunk 1]
```diff
@@ -1 +1 @@
...
...,"/Users/julianikolova/home-wall/src/js/api/HomeWallApi.js",[],"/Users/julianikolova/home-wall/src/js/components/HomeWalls.js",[],"/Users/julianikolova/home-wall/src/js/components/HomeWall.js",[],"/Users/julianikolova/home-wall/src/js/components/nav/NavBar.js",[],"/Users/julianikolova/home-wall/src/js/containers/ProblemEditorContainer.js",[],"/Users/julianikolova/home-wall/src/js/components/nav/MenuDrawer.js",[],"/Users/julianikolova/home-wall/src/js/components/views/ProblemView.js",[],"/Users/julianikolova/home-wall/src/js/components/views/ProblemList.js",[],"/Users/julianikolova/home-wall/src/js/components/login/Login.js",[],"/Users/julianikolova/home-wall/src/js/components/views/ProblemEditor.js",[],"/Users/julianikolova/home-wall/src/js/utils/RayCasting.js",[],"/Users/julianikolova/home-wall/src/js/utils/Holds.js",[],"/Users/julianikolova/home-wall/src/js/utils/Grades.js",[],"/Users/julianikolova/home-wall/src/js/components/Board.js",[],"/Users/julianikolova/home-wall/src/js/components/buttons/AddProblemButton.js",[],{"ruleId":"85","replacedBy":"86"},{"ruleId":"87","replacedBy":"88"},"no-native-reassign",["89"],"no-negated-in-lhs",["90"],"no-global-assign","no-unsafe-negation"]
```
Comment (julesnikolova) [inline .eslintcache]: What is this and why is it generated?

Comment (scottrw93) [reply .eslintcache]: It for eslint to check style of code is correct. Shouldnt be checked in, ill remove and to .gitignore file later

File: src/js/components/views/ProblemList.js [hunk 2]
```diff
@@ -49,6 +49,7 @@ const ProblemList = ({
     name: null,
     grade: null,
     author: null,
+    createdAt: null
```
Comment (scottrw93) [inline src/js/components/views/ProblemList.js]: Remove this line

File: src/js/components/views/ProblemList.js [hunk 3]
```diff
@@ -76,7 +77,11 @@ const ProblemList = ({
             !filters.grade ||
             (grade <= filters.grade.lte && grade >= filters.grade.gte),
         )
-        .map(({ uuid, holds, name, grade, author }) => (
+        .filter(
```
Comment (scottrw93) [inline src/js/components/views/ProblemList.js]: Remove. These are still a WIP, the intention is to limit the problems you see on screen e.g between 6b and 7a, starting with `Emoji` etc.

**Pull request merged**